packages/code-studio/src/styleguide/Grids.jsx

```diff
@@ -5,7 +5,7 @@ import {
   MockTreeGridModel,
   ThemeContext,
 } from '@deephaven/grid';
-import IrisGrid from '@deephaven/iris-grid/dist/IrisGrid';
+import IrisGrid from '@deephaven/iris-grid';
```

Comment:
I think there's preference for named exports over defaults in ESM now. Supposedly the defaults were mostly for CJS compatibility and named only make some difference to potential tree shaking and refactoring. Also auto importing works better w/ named imports.

Could be a separate ticket if it's something we want to enable
[eslint rule](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-default-export.md)
[some reasons to avoid default](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad)
[some more reasons to avoid default](https://rajeshnaroth.medium.com/avoid-es6-default-exports-a24142978a7a)

Looks like 1 exception is on code splitting in React

Reply:
Created a ticket for this: https://github.com/deephaven/web-client-ui/issues/585